src/test/resources/dummy4j/en/finance/bank_account.yml

```diff
@@ -0,0 +1,94 @@
+en:
+  finance:
+    bank_account:
+      number_structure:
+        AD: "####################"
```

Comment:
Do we need *all* countries in the test .yml? One should suffice, in theory.

Reply:
we test for enum coverage